packages/common/package.json

```diff
@@ -18,7 +18,8 @@
     "orm"
   ],
   "peerDependencies": {
-    "reflect-metadata": "^0.1.13"
+    "reflect-metadata": "^0.1.13",
+    "class-transformer": "*"
```

Comment:
Shouldn't class-transformer be moved from dependencies to devDependencies with this change?

Reply:
This would be a breaking change as every consumer would be required to provide it's own copy of class-transformer. This only changes the priority :)

Reply:
Agree with @Michsior14.